Sources/StreamChatUI/xibs/cells/TableViewCellWallePayBubbleIncoming.swift

```diff
@@ -282,12 +289,11 @@ public class TableViewCellWallePayBubbleIncoming: UITableViewCell {
             userInfo["recipientName"] = requestedUserName(raw: payload.extraData)
             userInfo["recipientUserId"] = requestedUserId(raw: payload.extraData)
             userInfo["requestedImageUrl"] = requestedImageUrl(raw: payload.extraData)
+            userInfo["paymentTheme"] = requestedThemeURL(raw: payload.extraData)
+            if let channelId = channelId {
+                userInfo["channelId"] = channelId.description
```

Comment:
@jitendra-simform  You can pass channelId directly from here as well.

Reply:
@ajay-simform  I think it's good if we pass only channel id as parameter, like other parameters, rather sending entire channel ID object.